MapboxSearch/base/src/main/java/com/mapbox/search/base/MetadataExt.kt

```diff
@@ -66,6 +73,20 @@ fun ParkingData.mapToCore() = CoreParkingData(
     reservedForDisabilities
 )
 
+fun CoreChildMetadata.mapToPlatform() = ChildMetadata(
+    mapboxId,
+    name,
+    category,
+    coordinates
+)
+
+fun ChildMetadata.mapToCore() = CoreChildMetadata(
+    category,
+    coordinates,
+    mapboxId,
+    name
+)
```

Comment:
Noticed that the order of these parameters in mapToPlatform and mapToCore are different. Is this by design or should these have the same order?

Reply:
This is because of the fields in the class, mapboxId is not allowed to be null:
```
public class ChildMetadata(
    public val mapboxId: String,
    public val name: String?,
    public val category: String?,
    public val coordinates: Point?
) 
```